.travis.yml

```diff
@@ -9,6 +9,8 @@ install:
   - go get -v github.com/mattn/goveralls
 
 script:
+  # This unencryptes envvars.bash.enc into envvars.bash so that we can run the tests
+  - openssl aes-256-cbc -K $encrypted_263604b5b445_key -iv $encrypted_263604b5b445_iv -in envvars.bash.enc -out envvars.bash -d
```

Comment:
Travis uses GitHub for authentication, so you should be able to log in with your own credentials per these instructions - http://docs.travis-ci.com/user/getting-started/#Step-one%3A-Sign-in

Once you're logged in, you can follow these instructions (linked from the README) to encrypt the file - http://docs.travis-ci.com/user/encrypting-files/